utils/yoke/src/yoke.rs

```diff
@@ -131,10 +125,6 @@ impl<Y: for<'a> Yokeable<'a>, C: StableDeref> Yoke<Y, C> {
 
     /// Construct a [`Yoke`] by yokeing an object to a cart. If an error occurs in the
     /// deserializer function, the error is passed up to the caller.
-    ///
-    /// Call sites for this function may not compile; if this happens, use
```

Comment:
issue: I would leave this comment in, mention "may not compile pre-1.61". And also to file a bug if they do not compile post 1.61